addons/hr/security/ir.model.access.csv

```diff
@@ -14,3 +14,5 @@ access_hr_plan_activity_type_hr_user,access_hr_plan_activity_type,model_hr_plan_
 access_hr_plan_hr_user,access_hr_plan_hr_user,model_hr_plan,group_hr_user,1,1,1,1
 access_hr_plan_wizard,access.hr.plan.wizard,model_hr_plan_wizard,hr.group_hr_manager,1,1,1,0
 access_hr_departure_wizard,access.hr.departure.wizard,model_hr_departure_wizard,hr.group_hr_user,1,1,1,0
+access_work_location_user,access_work_location_user,model_work_location,group_hr_user,1,0,0,0

```

Comment:
It is better tu use base.group_user to allow all employee to know location of college.